### PR TITLE
Task/sapnamysore/tlt 1870/mp unit tests

### PR DESCRIPTION
**Pull request by sapnamysore**
This PR contains imported unit tests for 
1. test_utils(skipped one test due to issue in loading fixture)
2. template_tags

Putting this PR out as we work on the pending stuff:
1. switch to sqlite3 
2. fix a  skipped test in test_utils
3. tests in test_views

### CODE REVIEW

File: manage_people/tests/test_template_tags.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from django.test import TestCase
+
+# from icommons_common.models import UserRole
```
Comment (eparker71) [inline manage_people/tests/test_template_tags.py]: clean up commented code

File: manage_people/tests/test_template_tags.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+from manage_people.templatetags import manage_people_tags
+# from manage_people.models import ManagePeopleRole
```
Comment (eparker71) [inline manage_people/tests/test_template_tags.py]: same

File: manage_people/tests/test_utils.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    get_available_roles,
+    get_course_member_class,
+)
+
+
+class GetAvailableRolesTest(TestCase):
+    fixtures = [
+        'manage_people_role.json',
+        'user_role.json',
+        # 'school_allowed_role.json',
```
Comment (eparker71) [inline manage_people/tests/test_utils.py:22]: clean up commented code

File: manage_people/tests/test_template_tags.py (hunk 2 above)
Comment (sapnamysore) [inline manage_people/tests/test_template_tags.py]: Sure. Thanks!

**Comment by eparker71** [comment]
👍 Looks good

**Pull request merged**